SimPEG/Maps.py

```diff
@@ -372,7 +396,7 @@ def deriv(self, m, v=None):
             :rtype: numpy.array
             :return: derivative of transformed model
         """
-        deriv = np.ones([self.mesh.nC, 1])
+        deriv = sp.csr_matrix(np.ones([self.mesh.nC, 1]))
```

Comment:
this is silly (it is a dense sparse matrix!), but we are expecting to be able to multiply by it...